exercises/darts/description.md

```diff
@@ -14,4 +14,4 @@ In our particular instance of the game, the target rewards with 4 different amou
 
 The outer circle has a radius of 10 units (This is equivalent to the total radius for the entire target), the middle circle a radius of 5 units, and the inner circle a radius of 1. Of course, they are all centered to the same point (That is, the circles are [concentric](http://mathworld.wolfram.com/ConcentricCircles.html)) defined by the coordinates (0, 0).
 
-Write a function that given a point in the target (defined by its `real` cartesian coordinates `x` and `y`), returns the correct amount earned by a dart landing in that point.
+Write a function that given a point in the target (defined by its [real](https://www.mathsisfun.com/numbers/real-numbers.html) [cartesian coordinates](https://www.mathsisfun.com/data/cartesian-coordinates.html) `x` and `y`), returns the correct amount earned by a dart landing in that point.
```

Comment:
```suggestion
Write a function that given a point in the target (defined by its [cartesian coordinates](https://www.mathsisfun.com/data/cartesian-coordinates.html) `x` and `y`, where `x` and `y` are [real](https://www.mathsisfun.com/numbers/real-numbers.html)), returns the correct amount earned by a dart landing in that point.
```

Reply:
How about this? @petertseng @ErikSchierboom

Reply:
:+1

Reply:
any opinions on "real numbers" versus just "real"?

Reply:
How about leaving that out altogether and just use "Cartesian coordinates"  If they can pass the tests using not numbers at all, it would be mentor-worthy.

Reply:
That they're real numbers may be useful as a hint that the student needs to use floating point numbers.

Reply:
> any opinions on "real numbers" versus just "real"?

I'm fine with either 🤷

Reply:
Please explain how a test case with `-0.1` is gonna work with integers.

Reply:
Many languages allow for float comparison with integers.  As a value on a dart board, this value seems invalid, because if it is the distance from center, it does not make sense.  So I suppose I would need more information.

In Ruby, the test for where we land looks like this, for my current solution:

```ruby
  BOARD = {
    10 => 0..1,
     5 => 1..5,
     1 => 5..10
  }
```

Integers used all around, not floats or other numeric types found in the code.

But as a practice exercise, it is up to the student to solve this with whatever concepts they would like to explore, such as how floats can or can not work with integers in their language.

Reply:
That is poorly worded (the test for) but this is the object that is fundamentally used to test for the location being given a score.  How this is applied is in a different area, but it also involves no floats, though a float may be provided as a distance among other numeric types.

Reply:
Talking about this test case for example:  https://github.com/exercism/ruby/blob/95de4127a188821da217fe1e4bf4212dee841bd7/exercises/practice/darts/darts_test.rb#L36-L40

```ruby
  def test_near_the_centre
    skip
    darts = Darts.new(-0.1, -0.1)
    assert_equal(10, darts.score)
  end
```

So mentioning that the coordinates are real is useful, as it gives a hint that these values are allowed (and most likely floats) and need to be considered. Even if the solution doesn't explicitly require them.

Reply:
> Please explain how a test case with -0.1 is gonna work with integers.

Then my explanation would involve the phrase "just fine".  I am assuming the test that you have shown does not blow up if integers are compared in the language.

A deeper understanding of how it will work with integers very likely involves coercion to the base most common type, in this case, likely the equivalent integer in the form of a float.